frontend/src/interfaces/person.dto.ts

```diff
@@ -1,4 +1,4 @@
-export interface IPersonDTO {
+export interface IPersonProps {

```

Comment:
Could you please justify why did you rename the interface to IPersonProps?

Reply:
Because the name does not match the import that is used in person-card. I think it is better to change the name of the interface instead of the import name and the property name